src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -930,7 +928,8 @@ public void testRawGtCountAnnotation() {
         args.addReference(b37_reference_20_21)
                 .addVCF(reblockedGVCF)
                 .addOutput(output)
-                .add("keep-combined-raw-annotations", true)
```

Comment:
Sorry, just to be clear, you mean you'll revert this test to its original behavior, right? In that case, will you add another test for the keep-specified behavior?

Reply:
The original behavior of `--keep-combined-raw-annotations` set to true while `--keep-specified-raw-annotations` is not empty is no longer allowed due to the `mutex`. So I think I am not reverting this test to the original behavior. Does that make sense? This test sets `--keep-specified-raw-annotations` to `RawGtCount`

Reply:
Sorry, by "original behavior" I mean the current behavior of the test as it is in master. This test sets `--keep-combined-raw-annotations true` and hence additionally retains `RAW_MQandDP`.

In contrast, you modify the test in your branch so that only `RAW_GT_COUNT` is retained. This means the code path that is responsible for retaining `RAW_MQandDP` is no longer run in this test, at least (and is perhaps not run for this particular input test file in any test at all). Even though the original test does not explicitly check for any particular `RAW_MQandDP` output, this is still an effective drop in test coverage.

Probably not anything to sweat about, but that's all that I wanted to point out in my original comment. I'll leave it up to you to decide if `--keep-combined-raw-annotations true` is sufficiently covered by the other tests so that we don't have to worry about replacing the original test here.

Reply:
Oh! Sorry, I see what you're saying now. I think I originally added this test as a way of testing `RawGTCount`, rather than `--keep-combined-raw-annotations true`, so I overlooked that it did actually test the latter too. I do think the other tests sufficiently cover `--keep-combined-raw-annotations true` but I very much see how that is not clear from this change.